### PR TITLE
Feature/facility

### PR DESCRIPTION
**Pull request by amalaabraham**

### CODE REVIEW

File: src/facilities/dto/create-facility.dto.ts [hunk 1]
```diff
@@ -1 +1,40 @@
-export class CreateFacilityDto {}
+import { ApiProperty } from '@nestjs/swagger';
+import { IsNotEmpty, IsString } from 'class-validator';
+import { CreateFacilityStatus } from '../create-facility-status.enum';
+
+export class CreateFacilityDto {
+  @ApiProperty({ type: 'number' })
+  @IsNotEmpty()
+  typeid: number;
```
Comment (riginoommen) [inline src/facilities/dto/create-facility.dto.ts]:
```suggestion
  typeId: number;
```

File: src/facilities/dto/create-facility.dto.ts [hunk 2]
```diff
@@ -1 +1,40 @@
...
+  description: string;
+
+  @ApiProperty({ type: 'string' })
+  latitude: string;
+
+  @ApiProperty({ type: 'string' })
+  longitude: string;
+
+  @ApiProperty({ type: 'string' })
+  contact: string;
```
Comment (riginoommen) [inline src/facilities/dto/create-facility.dto.ts:33]: What is going to store here can you please provide a sample input

File: src/facilities/migrations/1611981680527-facilities.ts [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+            type: 'bigint',
+            isPrimary: true,
+            isGenerated: true,
+            generationStrategy: 'increment',
+          },
+          //   {
+          //     name: 'typeid',
+          //     type: 'bigint',
+          //     isNullable: true,
+          //   },
```
Comment (riginoommen) [inline src/facilities/migrations/1611981680527-facilities.ts]:
```suggestion
```

**Review by riginoommen — CHANGES_REQUESTED** [review]
Please check the comments

File: src/facilities/dto/create-facility.dto.ts (hunk 2 above)
Comment (amalaabraham) [reply src/facilities/dto/create-facility.dto.ts:33]:
```json
{
  "typeid": 1,
  "name": "theatre",
  "address": "megha arcade,power house road",
  "pincode": 686578,
  "description": "string",
  "latitude": "string",
  "longitude": "string",
  "contact": "string",
  "imageUrl": "http://www.google.com/arihant.png",
  "status": "Status:OPEN,CLOSE,IN_ACTIVE"
}
```

File: src/routes/routes.service.ts [hunk 4]
```diff
@@ -1,4 +1,17 @@
 import { Injectable } from '@nestjs/common';
+import { CreateRouteDto } from './dto/create-route.dto';
+import { routeRepository } from './routes.repository';
+import { InjectRepository } from '@nestjs/typeorm';
+import { Route } from './entities/route.entity';
```
Comment (athul) [inline src/routes/routes.service.ts:5]:
```suggestion
```

Analysis failed due to unused import. Resolving this will render the checks green 👌

**Review by athul — CHANGES_REQUESTED** [review]

**Pull request closed without merging**